config/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,247 @@
+<?xml version="1.0"?>
+<!--
+
+    Copyright 2017 The OpenTracing Authors
+
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+    in compliance with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software distributed under the License
+    is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+    or implied. See the License for the specific language governing permissions and limitations under
+    the License.
+
+-->
+<!DOCTYPE module PUBLIC
+  "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+  "http://checkstyle.sourceforge.net/dtds/configuration_1_3.dtd">
+
```

Comment:
Could you please also include original authors header https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/google_checks.xml#L6

```
<!--
    Checkstyle configuration that checks the Google coding conventions from Google Java Style
    ...
    Authors: Max Vetrenko, Ruslan Diachenko, Roman Ivanov.
 -->
```

Otherwise It looks good to me. I would like to also enable import order check but it can be done separatelly.

Reply:
header added